src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+            return valid[choice]
+        else:
+            logger.info("Please respond with 'yes' or 'no' " "(or 'y' or 'n').\n")
+
+
+class Command(BaseCommand):
+    help = """Load data for domains that are in transition
+    (populates transition_domain model objects)."""
+
+    def add_arguments(self, parser):
+        """Add our three filename arguments."""
+        parser.add_argument(
+            "domain_contacts_filename", help="Data file with domain contact information"
+        )
+        parser.add_argument(
+            "contacts_filename",
+            help="Data file with contact information",
+        )
+        parser.add_argument(
+            "domain_statuses_filename", help="Data file with domain status information"
+        )
+
+        parser.add_argument("--sep", default="|", help="Delimiter character")
+
+        parser.add_argument("--debug", action=argparse.BooleanOptionalAction)
+
+        parser.add_argument(
+            "--limitParse", default=0, help="Sets max number of entries to load"
+        )
+
+        parser.add_argument(
+            "--resetTable",
+            help="Deletes all data in the TransitionDomain table",
+            action=argparse.BooleanOptionalAction,
+        )
+
+    def handle(  # noqa: C901
+        self,
+        domain_contacts_filename,
+        contacts_filename,
+        domain_statuses_filename,
+        **options,
+    ):
+        """Load the data files and create the DomainInvitations."""
+        sep = options.get("sep")
+
+        if options.get("resetTable"):
+            confirmReset = query_yes_no(
+                f"""
+                {termColors.FAIL}
+                WARNING: Resetting the table will permanently delete all
+                the data!
+                Are you sure you want to continue?{termColors.ENDC}"""
+            )
+            if confirmReset:
+                logger.info(
+                    f"""{termColors.WARNING}
+                ----------Clearing Table Data----------
+                (please wait)
+                {termColors.ENDC}"""
+                )
+                TransitionDomain.objects.all().delete()
+
+        debug_on = options.get("debug")
+        debug_max_entries_to_parse = int(
+            options.get("limitParse")
+        )  # set to 0 to parse all entries
+
+        self.print_debug_mode_statements(debug_on, debug_max_entries_to_parse)
+
+        # STEP 1:
+        # Create mapping of domain name -> status
+        domain_status_dictionary = self.get_domain_user_dict(
+            domain_statuses_filename, sep
+        )
+
+        # STEP 2:
+        # Create mapping of userId  -> email
+        user_emails_dictionary = self.get_user_emails_dict(contacts_filename, sep)
+
+        # STEP 3:
+        # TODO:  Need to add logic for conflicting domain status
+        # entries
+        # (which should not exist, but might)
+        # TODO: log statuses found that don't map to the ones
+        # we have (count occurences)
```

Comment:
Were these TODOs for another upcoming ticket or did you forget to remove these guys?

Reply:
Oh my...I should have deleted those.  Yes, the logic is there. I will do that.